packages/di/src/common/decorators/lazyInject.ts

```
@@ -10,42 +34,26 @@
  */
-export function LazyInject(
-  key: string,
-  resolver: () => any,
-  {optional = false, packageName = resolver.toString()}: {optional?: boolean; packageName?: string} = {}
-): PropertyDecorator {
-  return (target: any, propertyKey: string | symbol): any | void => {
-    let bean: any, token: any;
+export function LazyInject(resolver: () => Promise<{default: unknown}>): PropertyDecorator;
+export function LazyInject(key: string, resolver: () => Promise<{default: unknown}>): PropertyDecorator;
+export function LazyInject(...args: any[]): PropertyDecorator {
+  let resolver = mapOptions(args);
```

Biome

[error] This let declares a variable that is only assigned once.

'resolver' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)